apps/admin-gui/src/app/shared/components/dialogs/assign-service-to-resource-dialog/assign-service-to-resource-dialog.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="{{theme}}">
+  <h1 mat-dialog-title>{{'DIALOGS.ASSIGN_SERVICE_TO_RESOURCE.TITLE' | translate}}</h1>
+  <app-debounce-filter
+    placeholder="{{'DIALOGS.ASSIGN_SERVICE_TO_RESOURCE.FILTER_DESCRIPTION' | translate}}"
+    (filter)="applyFilter($event)">
+  </app-debounce-filter>
+  <mat-spinner *ngIf="loading" class="mr-auto ml-auto"></mat-spinner>
+  <app-services-list
+    (page)="pageChanged($event)"
+    [pageSize]="pageSize"
+    *ngIf="!loading"
+    [services]="filteredServices"
```

Comment:
same here with filtering of the services
[service]="services"
[fileringValue]="filteringValue"